### ISSUE TITLE
Failed tasks (status=ERROR) should show stderr more easily

### ISSUE THREAD

**Issue by kdemarest** [appsoma/welder#180]
This is a programmer productivity enhancement.

There are two places where you'd like to see a failed task's stderr without having to click in:

1) on the /runs page, clicking the ERROR mark should reveal what Task(s) failed and their stderr

![image](https://cloud.githubusercontent.com/assets/2153005/8575244/94cc3dd6-2560-11e5-842b-6aa41267b83b.png)

2) on the /runs/{myrun} page, any task with ERROR status could show stderr to the right of "View Details"

![image](https://cloud.githubusercontent.com/assets/2153005/8575262/b037c914-2560-11e5-8789-ddc6a9f9d55f.png)

**Comment by AgustinCB** [comment]
https://github.com/appsoma/welder/pull/200

First point I see adds a lot of unnecessary complexity on the page. The amount of clicks to see the actual stdout error is the same than if you just go to the run details page and see it.

**Comment by kdemarest** [comment]
yes, agreed. Perhaps hovering it brings up a tooltip with the task ID and the first 40 characters of the error? Or maybe we just don't do that.
